src/openms/source/ANALYSIS/ID/IDBoostGraph.cpp

```diff
@@ -191,15 +194,16 @@ namespace OpenMS
       int i = 0;
 
       #pragma omp parallel for
-      for (const Graph& fg : ccs_)
+      for (auto g_it = ccs_.begin() ; g_it < ccs_.end(); ++g_it)
```

Comment:
I dont think that works on Windows, for this you need OpenMP 3.0 : https://stackoverflow.com/questions/2513988/iteration-through-std-containers-in-openmp  @cbielow is that correct?